README.md

```diff
@@ -99,28 +99,18 @@ If you do not wish to use the Mycroft Home service, before starting Mycroft for
 }
 ```
 
-Mycroft will then be unable to perform speech-to-text conversion, so you'll need to set that up as well, using one of the [STT engines Mycroft supports](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/stt-engine).
-
-You may insert your own API keys into the configuration files listed above in <b>Configuration</b>.  For example, to insert the API key for the Weather skill, create a new JSON key in the configuration file like so:
-
-```
-{
-  // other configuration settings...
-  //
-  "WeatherSkill": {
-    "api_key": "<insert your API key here>"
-  }
-}
-```
-
 ### API Key Services
 
-These are the keys currently used in Mycroft Core:
+The Mycroft backend provides access to a range of API keys for specific services. Without pairing with the Mycroft backend, you will need to add your own API keys, install a different Skill or Plugin to perform that function, or not have access to that functionality.
+
+These are the keys currently used in Mycroft Core through the Mycroft backend:
 
 - [STT API, Google STT, Google Cloud Speech](http://www.chromium.org/developers/how-tos/api-keys)
+  - [A range of STT services](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/stt-engine) are available for use with Mycroft.
```

Comment:
Is this the intended indentation level? It looks a bit weird in the preview

Reply:
Yeah it was. It does look a little weird, but it's part of the point above...
I'm just going to go with it for the moment. 
The whole README could probably do with a refresh...

Thanks for the eagle eyes though :)